plugins/flex-basis/index.js

```diff
@@ -0,0 +1,16 @@
+module.exports = () => ({ addUtilities, e, theme, variants }) => {
+  const pluginConfig = theme('flexBasis', {})
+  const pluginVariants = variants('flexBasis', [])
+
+  const utilities = Object.keys(pluginConfig).map(key => {
+    return {
+      [`.${e(`flex-basis-${key}`)}`]: {
+        'flex-basis': pluginConfig[key]
+      }
+    }
+  })
+
+  addUtilities(utilities, {
+    variants: pluginVariants
```

Comment:
seeing differences in how this is written in different plugins. Is there a reason it is set to a constant in this one but in other plugins we do:

```js
variants: variant('flexBasis', [])
```

it becomes a little confusing for someone who hasn't seen it before and may lead to "wait, how do I do this?" type of questions. tl;dr if possible let's settle on one way and do it that way in all of these.

Reply:
Actually either way we dont need the variants key at all here:
```js
addUtilities(utilities, variants('flexBasis', [])
```

Reply:
Saving plugin config and variants to a constant is a pattern I picked up from somewhere. I thought it was a core TW feature, but I can't find it in the codebase. Must have been from somewhere else. @gregkohn streamlined things in #2 so I'll leave this as is so there are no conflicts.